### DISCUSSION TITLE
June Personal Web - Fetch Current Script Tag Attributes in Javascript

### DISCUSSION THREAD

**Discussion by giscus** [ClavinJune/ClavinJune.github.io#23] [General]
# June Personal Web - Fetch Current Script Tag Attributes in Javascript

fetching current script tag attributes in Javasvript

https://clavinjune.dev/en/blogs/fetch-current-script-tag-attributes-in-javascript/

**Comment by jcoady** [comment]
How do I determine my   CATEGORY ID used in  data-category-id="[ENTER CATEGORY ID HERE]" from giscuss. I created a CATEGORY on giscuss but I don't know the CATEGORY ID.

**Reply by clavinjune** [reply]
Hi @jcoady, You can check it [here](https://giscus.app/), don't forget to input your repository name

<img width="854" alt="image" src="https://github.com/clavinjune/clavinjune.dev/assets/24659468/48b8c100-57f6-4832-bd1f-78eb49ffda10">
